components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/internal/handler/TokenEventHookHandler.java

```diff
@@ -152,4 +162,26 @@ private void publishEvent(String tenantDomain, Channel tokenChannel, String even
         EventHookHandlerDataHolder.getInstance().getEventPublisherService()
                 .publish(securityEventTokenPayload, eventContext);
     }
+
+    private String resolveApplicationKey(EventData eventData) {
+
+        if (eventData == null) {
+            return null;
+        }
+
+        if (String.valueOf(eventData.getProperties().get(IdentityEventConstants.EventProperty.CONSUMER_KEY)) != null) {
```

Comment:
This condition will always be true because String.valueOf() never returns null - it returns "null" as a string when the input is null. This makes the else-if branch unreachable.